### PR TITLE
Update Readme

### PR DESCRIPTION
**Pull request by valouvaliavlo**
Thank you for sending your pull request. 
But first, have you included unit tests, and is your code PEP8 conformant? [More details](https://github.com/Drakkar-Software/OctoBot/blob/dev/CONTRIBUTING.md)

## Summary
Explain in one sentence the goal of this PR

Solve the issue: #___

## Changelog

  - _____
  - _____
  - _____

## What's new?
*Explain in details what this PR solve or improve.*

### CODE REVIEW

**Review by GuillaumeDSM — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -60,7 +60,15 @@ docker run -itd --name OctoBot -p 80:5001 -v $(pwd)/user:/octobot/user -v $(pwd)
 ```
 And then open [http://localhost](http://localhost).
 
-#### [With python sources](https://docs.octobot.online/installation/with-docker)
+#### [With pip](https://docs.octobot.online/installation/with-pip)
+
+In short :
+```
+pip install OctoBot>=0.4.0b14
```
Comment (GuillaumeDSM) [inline README.md:67]: actually, if we use `pip install -U OctoBot` we don't need the version

Comment (Herklos) [reply README.md:67]: Does it work when it is not already installed ?

Comment (valouvaliavlo) [reply README.md:67]: it's 0.3.9 which is installed without version of specified

Comment (valouvaliavlo) [reply README.md:67]: https://pypi.org/project/OctoBot/

Comment (Herklos) [reply README.md:67]: Right, that's why I said we''ll have to update this line at each update. I don' t know an option to install the latest pre-release version

Comment (GuillaumeDSM) [reply README.md:67]: Then I think `OctoBot>=0.4.0b14` is enough not to install 0.3.9, we don't need to update this number alongside the version number in new versions

Comment (Herklos) [reply README.md:67]:
```
~ $ pip3 install -U OctoBot
Collecting OctoBot
  Using cached OctoBot-0.3.9-py2.py3-none-any.whl (682 kB)
```

I agree OctoBot>=0.4.0b14 can do the job. I think we have to stop using pre release version form.

Comment (GuillaumeDSM) [reply README.md:67]: right

**Pull request merged**